src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -182,6 +184,9 @@ NTFS accessibility value | Netex accessibility value
 Modes](#netex-transport-modes)).  For each mode, a `StopPlace` of type `LMO` is
 created.  An additionnal `LMU` is created to regroup them.
 
+Station entrances/exits (stops with `location_type` = 3) are defined in a 
+monomodal `StopPlace` whose `ParentSiteRef` matches the stop's `parent_station`.
```

Comment:
@prhod 
It will be rather on multimodal.
No interest in putting it on monomodal. And ok with Netex-fr specs.
```
Station entrances/exits (stops with location_type = 3) are defined in a multimodal StopPlace to which they belong.
```